example-ws/src/index.ts

```diff
@@ -17,7 +17,9 @@ const sub = (a: number, b: number): number => {
 wss.on("connection", function connection(ws) {
   console.log("new ws connection");
 
-  const rpc: IRpc = new RpcExtensionWebSockets(ws);
+  // logger is optional third parameter, implementing interface IChildLogger:
+  // https://github.com/SAP/vscode-logging/blob/master/packages/types/api.d.ts#L17
+  const rpc: IRpc = new RpcExtensionWebSockets(ws /*, logger*/);
```

Comment:
```suggestion
  const rpc: IRpc = new RpcExtensionWebSockets(ws);
```

Reply:
why ?

Reply:
up to you, but for me it is redundant. the constructor signature is well defined in class definition

Reply:
done